main.go

```diff
@@ -172,6 +176,10 @@ func main() {
 		setupLog.Error(err, "unable to create webhook", "webhook", "GCPMachineTemplate")
 		os.Exit(1)
 	}
+	if err = (&infrav1exp.GCPManagedControlPlane{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
nit, we need to remove this as well until the feature flag is enabled.

Reply:
done